### ISSUE TITLE
why we set norm_eval=True in object detection task ?

### ISSUE THREAD

**Issue by johnson-magic** [open-mmlab/mmdetection#5642]
type='ATSS',
    backbone=dict(
        type='ResNet',
        depth=50,
        num_stages=4,
        out_indices=(0, 1, 2, 3),
        frozen_stages=1,
        norm_cfg=dict(type='BN', requires_grad=True),
        norm_eval=True,
        style='pytorch',
        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),

**Comment by PeterVennerstrom** [comment]
Frozen batchnorm parameters from Imagenet pretraining can outperform batchnorm trained on downstream tasks like object detection trained on smaller datasets.
